config.ru

```diff
@@ -0,0 +1,25 @@
+# This file is used by Rack-based servers to start the application.
```

Comment:
You should follow the instructions under `Rails` instead of `Rack`

https://github.com/cyu/rack-cors